src/main/java/seedu/address/storage/XmlSerializableAddressBook.java

```diff
@@ -58,6 +63,20 @@ public XmlSerializableAddressBook(ReadOnlyAddressBook src) {
     }
 
     @Override
+    public ObservableList<ReadOnlyPerson> getBlacklistedPersonList() {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.